### ISSUE TITLE
Dapper Deserialization issue of Nullable Decimal also related to LiteLib NumericAffinity

### ISSUE THREAD

**Issue by codephined** [unosquare/litelib#25]
Deserializing a nullable `decimal` throws a cast exception in Dapper.  It's reported several times on their issues page.
The issue arose when a nullable decimal was saved without precision (for instance, `123.00`). The value is saved to SQLite as `123` and then Dapper attempts to convert it into an `Int32`/`Int64`.

I created a workaround for this error in Dapper, but I also had to change something in LiteLib.
https://github.com/unosquare/litelib/blob/72309fda84ddaac7f82540652d16ed9c5d43cef5/src/Unosquare.Labs.LiteLib/Extensions.cs#L26
from
````
            {typeof(decimal), NumericAffinity},
````
to
````
           private const string RealAffinity = "REAL";
...
            {typeof(decimal), RealAffinity},
````

Now decimal values without precision are being saved as `123.0`.
Thanks

**Comment by geoperez** [comment]
Let me integrate and test your suggestion. Thanks!

**Comment by geoperez** [comment]
I just published a new nuget v0.23.0

**Comment by codephined** [comment]
Thanks!
